lldb/include/lldb/Core/Debugger.h

```diff
@@ -568,10 +569,22 @@ class Debugger : public std::enable_shared_from_this<Debugger>,
 
   static void ReportSymbolChange(const ModuleSpec &module_spec);
 
-  void
+  /// DEPRECATED. Use AddDestroyCallback and RemoveDestroyCallback instead.
+  /// Clear all previously added callbacks and only add the given one.
```

Comment:
I would give a reason for the deprecation, like:

DEPRECATED: We used to only support one Destroy callback.  Now that we support Add and Remove, you should only remove Destroy callbacks that you Add-ed.